docs/Networking.asciidoc

```diff
@@ -316,6 +325,8 @@ wicked ifup br1
 
 * REBOOT
 
+NOTE: Also possible to switch from YaST.
```

Comment:
How about "It is also possible to switch using YaST"?

Reply:
Done